### DISCUSSION TITLE
Listing fields control number of authors to display

### DISCUSSION THREAD

**Discussion by drganghe** [quarto-dev/quarto-cli#6223] [Feature Requests]
### Description

Papers with large number of authors are outsized when rendering in the same row with other items. Not sure if there is a way to control number of authors to display, for example, first (or other numbers of) authors, et al. It will still show the full list of authors when clicking the full article.

**Comment by dragonstyle** [comment]
This is a nice suggestion let me have a look!

**Comment by mcanouil** [comment]
Closing in favour of https://github.com/quarto-dev/quarto-cli/issues/6245
